src/v3/src/components/Widget/index.tsx

```diff
@@ -101,14 +101,12 @@ export const Widget: FunctionComponent<WidgetProps> = (widgetProps) => {
   // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
-  const handleError = (transaction: IdxTransaction | undefined, error: unknown) => {
+  const handleError = (error: unknown) => {
     // TODO: handle error based on types
     // AuthApiError is one of the potential error that can be thrown here
     // We will want to expose development stage errors from auth-js and file jiras against it
     setAuthApiError(error as AuthApiError);
     console.error(error);
-    // error event
-    events?.afterError?.(transaction ? getEventContext(transaction) : {}, error);
```

Comment:
Removing the `afterError `event entirely from this `handleError` function because we only want to call this event hook after form errors, not API errors. Other PRs will add this back in the right places.